src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -2820,6 +2820,120 @@ class TezosDatabaseOperationsTest
       )
     }
 
+    "should aggregate with datePart aggregation" in {
+      val oneDay = 3600 * 24 * 1000
+      val feesTmp = List(
+        FeesRow(0, 2, 4, new Timestamp(0), "kind"),
+        FeesRow(0, 4, 8, new Timestamp(1 + oneDay), "kind"),
+        FeesRow(0, 3, 4, new Timestamp(2 + oneDay), "kind"),
+        FeesRow(0, 3, 4, new Timestamp(3 + oneDay), "kind"),
+        FeesRow(0, 3, 4, new Timestamp(1 + oneDay * 2), "kind"),
+        FeesRow(0, 3, 4, new Timestamp(2 + oneDay * 2), "kind")
+      )
+
+      val aggregate = List(
+        Aggregation("timestamp", AggregationType.datePart, None, Some("YYYY-MM-DD")),
+        Aggregation("medium", AggregationType.count, None, None)
+      )
+
+      val populateAndTest = for {
```

Comment:
I don't like when preparing data for a test mixes with checking them, the usual separation of 'given` and `when` is easier to read.

Reply:
I've started adding tests trying to stay with the convention. We could refactor this, but I would prefer creating new ticket for that and refactor whole file.